frame/encode.go

```diff
@@ -28,9 +29,15 @@ func encodeValue(s string) []byte {
 	return buf.Bytes()
 }
 
+// Reduce one allocation on copying bytes to string
```

Comment:
How big the overhead the extra allocation represents?
Also a while I like keeping short names I'd go with something more descriptive

Reply:
somehow I missed your replies, merging this now